Spec/Auth.swift

```diff
@@ -2091,61 +2091,283 @@ class Auth : QuickSpec {
             }
 
             // RSA10j
-            it("should supersede any configured params and options when TokenParams and AuthOptions were provided") {
-                let options = AblyTests.commonAppSetup()
-                options.clientId = "client_string"
-                let rest = ARTRest(options: options)
+            context("when TokenParams and AuthOptions are provided") {
 
-                let tokenParams = ARTTokenParams(clientId: options.clientId)
-                let defaultTtl = tokenParams.ttl
-                let defaultCapability = tokenParams.capability
+                it("should supersede configured AuthOptions (using key) even if arguments are empty") {
```

Comment:
This should say even argument objects are empty as saying arguments are empty makes it sound like if you don't pass in arguments as opposed to passing in empty objects